docs/how-to-guides/font-awesome-for-node-images.md

```diff
@@ -0,0 +1,48 @@
+---
+id: font-awesome-for-node-images
+title: Use Font Awesome icons for node images
+sidebar_label: Use Font Awesome for node images
+---
+
+[![Related - Tutorial](https://img.shields.io/static/v1?label=Related&message=Tutorial&color=008a00&style=for-the-badge)](/tutorials/style-your-graphs-in-memgraph-lab.md) [![Related - Blog
+Post](https://img.shields.io/static/v1?label=Related&message=Blog%20post&color=9C59DB&style=for-the-badge)](https://memgraph.com/blog/how-to-style-your-graphs-in-memgraph-lab)
+
+[Font Awesome](https://fontawesome.com/) is a popular icon library. You can't
+use those icons directly or SVG file as background images, but there is a
+workaround. Memgraph Lab supports the following image formats: `png`, `jpeg`,
+`gif` (static, not dynamic) and `webp`.
```

Comment:
I would maybe expand on this so it is clear what are why trying to achieve, what's the issue, and how to overcome it.

So something like (I wrote very non-documentation ready, so adjust to your own style):

If you ever tried to use a font awesome icon as a background image for a node, you might have noticed you were not able to do that by using the icon directly, because it is in SVG format and Lab doesn't support the SVG format. But Lab does support PNG, JPEG, GIF and WEBP formats, so there is a workaround for this problem.